### DISCUSSION TITLE
A few ideas from my VHF perspective

### DISCUSSION THREAD

**Discussion by LAwmand** [magicbug/Cloudlog#1563] [Ideas / Feature Requests]
My first post: I'm only qrv 2m and 23cm 

1. In propagation modes add "Aircraft Scatter".

2. On a custom map dates search could other search criteria be added such as mode to narrow down qso's such as "mode" or "propagation type"? So, for example I could just see my meteor scatter contacts in August rather than seeing everything.

3. The map produced in custom maps is wide and narrow, could this be expandable for better resolution over a wider north south perspective (maybe it can, but I couldn't see it).

David G0LBK

**Comment by magicbug** [comment]
> In propagation modes add "Aircraft Scatter".

This was recently added to the ADIF Spec so will be added very soon

> On a custom map dates search could other search criteria be added such as mode to narrow down qso's such as "mode" or "propagation type"? So, for example I could just see my meteor scatter contacts in August rather than seeing everything.

Currently you cannot but I think its a fair suggestion for adding and would be very useful

> The map produced in custom maps is wide and narrow, could this be expandable for better resolution over a wider north south perspective (maybe it can, but I couldn't see it).

This is tricky due to the limitations of the web browser, you can move the map around but the height isn't changeable yet.

**Comment by AndreasK79** [comment]
Check out my wide custom themes here: https://github.com/AndreasK79/cloudlog_wide_custom_themes
The maps adjust according to the vertical size of your browser. Note that you need the station_logbooks branch to use these.

**Reply by magicbug** [reply]
Davids hosted by me, do you have any screenshots of your custom themes @AndreasK79

**Reply by AndreasK79** [reply]
Well then, it's not that easy to use them. No screenshots, as they look the same, except they are wider. They are adapted from the original themes.

What's different from the regular themes in Cloudlog?

It's wider
The popup dialog is wider
The scrollbar is themed
The height of the dashboard map, gridsquare map and logbook map is higher

**Reply by AndreasK79** [reply]
Regular qso dialog:
![image](https://user-images.githubusercontent.com/6977712/184920187-90c79255-acc7-4df9-b9f1-1df44a320d7e.png)
Wide qso dialog:
![image](https://user-images.githubusercontent.com/6977712/184920257-92bb0941-eaaf-4eec-b7c7-f1d488022d77.png)

**Reply by AndreasK79** [reply]
Regular dashboard:
![image](https://user-images.githubusercontent.com/6977712/184921750-7c612126-a318-48f6-8c44-814a3d550ff4.png)
Wide dashboard:
![image](https://user-images.githubusercontent.com/6977712/184921858-6ad3dfea-7d98-4315-8121-3c741f1df4d9.png)
